project/Dependencies.scala

```diff
@@ -1,28 +1,34 @@
 import sbt._
 
 object Version {
-  val spark        = "1.4.0"
+  val spark             = "1.4.0"
 }
 
 object Library {
-  val sparkCore      = "org.apache.spark"  % "spark-core_2.11" % Version.spark
-  val scalaTest      = "org.scalatest" % "scalatest_2.11" % "2.2.4"
-  val adam = "org.bdgenomics.adam" % "adam-core_2.11" % "0.17.0"
-  val cli = "org.bdgenomics.utils" % "utils-cli_2.11" % "0.2.2"
-  val utils = "org.bdgenomics.utils" % "utils-misc_2.11" % "0.2.2"
-  val algebird = "com.twitter" % "algebird-core_2.11" % "0.10.2"
+  val sparkCore         = "org.apache.spark" %% "spark-core" % Version.spark
+  val sparkMLlib        = "org.apache.spark" %% "spark-mllib" % Version.spark
```

Comment:
I need to do sliding over the contig's fragments for the reference FASTA files, and there is an implementation of sliding in RDDFunctions which is part of MLlib. This is the only reason to depend on MLlib. There are other ways to do the sliding, but I am sure they won't be so efficient as the implementation in RDDFunctions.